samples/predict-text-sentiment-analysis.js

```diff
@@ -74,7 +74,7 @@ async function main(text, endpointId, project, location = 'us-central1') {
     }
   }
   predictTextEntityExtraction();
```

Comment:
Can you fix this as well (should be sentiment analysis here too)?

Reply:
Done.